src/app/models/payment.server.model.ts

```diff
@@ -107,6 +108,63 @@ const PaymentSchema = new Schema<IPaymentSchema, IPaymentModel>(
         },
       ],
     },
+
+    payment_fields_snapshot: {
```

Comment:
I'm thinking that we should set `payment_fields_snapshot` to be exactly `Form['payment_fields']` so that we only have one single source of truth (which is from Form`

Reply:
yes good point! fixed here [752654e](https://github.com/opengovsg/FormSG/pull/6591/commits/752654e4a54fdf06b478f6b8bbf8ef4c328f4a35)